certbot/ocsp.py

```diff
@@ -95,15 +94,20 @@ def determine_ocsp_server(self, cert_path):
 def _translate_ocsp_query(cert_path, ocsp_output, ocsp_errors):
     """Parse openssl's weird output to work out what it means."""
 
-    if not "Response verify OK" in ocsp_errors:
+    pattern = r"{0}: (WARNING.*)?good".format(cert_path)
```

Comment:
There's also [WARNING: no nonce in response](http://osxr.org:8080/openssl/source/apps/ocsp.c#0884) though maybe that shows up differently in the output?